test/renderer_test.rb

```diff
@@ -4,24 +4,24 @@ class RendererTest < NicePartials::Test
   test "render basic nice partial" do
     render("basic") { |p| p.content_for :message, "hello from nice partials" }
 
-    assert_rendered "hello from nice partials"
+    assert_text "hello from nice partials"
   end
 
   test "render nice partial in card template" do
     render(template: "card_test")
 
-    assert_rendered "Some Title"
-    assert_rendered '<p class="text-bold">Lorem Ipsum</p>'
-    assert_rendered "https://example.com/image.jpg"
+    assert_text "Some Title"
+    assert_css "p", class: "text-bold", text: "Lorem Ipsum"
+    assert_css("img") { _1["src"] == "https://example.com/image.jpg" }
```

Comment:
Should we swap to `assert_equal` here?

```suggestion
    assert_css("img") { assert_equal "https://example.com/image.jpg", _1["src"] }
```

Reply:
For this case, yes, let's do that.

The Capybara blocks expect a predicate, since it'll evaluate on each `img` element to determine if it's a match. An `assert_equal` call will raise instead of return `false`.

Since this page should only ever include a single `<img>`, and that element should only have a particular `[src]` attribute, this change feels appropriate.

Reply:
Oh, I didn't know that! Anytime I spot `==` in tests I'm thinking it's because we forgot to use `assert_equal` so I find the `assert_css` API a little confusing 😄